<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.1">
    <title>Links for odoo12-addon-repair-quality-control-issue</title>
  </head>
  <body>
    <h1>Links for odoo12-addon-repair-quality-control-issue</h1>
<a href="https://files.pythonhosted.org/packages/1b/81/65f8688fc99783d75834226c7b5bd47d0e12d27c36e985cf6b7539e1b420/odoo12_addon_repair_quality_control_issue-12.0.1.0.0.99.dev3-py3-none-any.whl#sha256=83451c2e6c785e3e15623b13d8edc6150cfdb9061a86bd3b1e5f0eb1a92bddf3" data-requires-python="&gt;=3.5" >odoo12_addon_repair_quality_control_issue-12.0.1.0.0.99.dev3-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/c2/8e/6d770c59bf003911d92ee67bdd8f6c69bc4386a35a0c62451e6aafb394d2/odoo12_addon_repair_quality_control_issue-12.0.1.0.1-py3-none-any.whl#sha256=8d4e9c3be33ca526bb8c5a85d8cfe1ecf38543babe6ed0c0390184f75f038966" data-requires-python="&gt;=3.5" >odoo12_addon_repair_quality_control_issue-12.0.1.0.1-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/7e/4f/9ea9b362659c438467d39ced3bf82d1fba7a1a49e1697ab991ac11583ee0/odoo12_addon_repair_quality_control_issue-12.0.1.0.1.99.dev2-py3-none-any.whl#sha256=76f27d6f5cd32261a77b018bb215409b66e643e96603662123564149dc08a6aa" data-requires-python="&gt;=3.5" >odoo12_addon_repair_quality_control_issue-12.0.1.0.1.99.dev2-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/86/43/0c0e3ae49fb2a082cd6cb7f480f74cd6dea353cb97494b6dda9f4be7d1b9/odoo12_addon_repair_quality_control_issue-12.0.1.0.1.99.dev3-py3-none-any.whl#sha256=32099da6757a778561aad2494e57d719777c82128acf87931173eb16afe37b4e" data-requires-python="&gt;=3.5" data-dist-info-metadata="sha256=1ffd14f8dc8fad8547c3445fe0c5b55603f09b34544af126276ac65c4316cdce" data-core-metadata="sha256=1ffd14f8dc8fad8547c3445fe0c5b55603f09b34544af126276ac65c4316cdce">odoo12_addon_repair_quality_control_issue-12.0.1.0.1.99.dev3-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/f5/50/f0e89605022a2f58b3db55a39804fcba0635fdd53824753222b2c9b9077e/odoo12_addon_repair_quality_control_issue-12.0.1.0.1.99.dev4-py3-none-any.whl#sha256=b22a824f21206e51b87a0c20a64f94128ea708daf81a5ff92e9a801618db5628" data-requires-python="&gt;=3.5" data-dist-info-metadata="sha256=ce081cbe39e9c5b4696c4f18845eb169ddbfcdf086766f6b2871014ec6c47cfb" data-core-metadata="sha256=ce081cbe39e9c5b4696c4f18845eb169ddbfcdf086766f6b2871014ec6c47cfb">odoo12_addon_repair_quality_control_issue-12.0.1.0.1.99.dev4-py3-none-any.whl</a><br />
</body>
</html>
<!--SERIAL 20116531-->